packages/react-components/react-progress/src/components/ProgressBar/ProgressBar.test.tsx

```diff
@@ -47,4 +53,41 @@ describe('ProgressBar', () => {
     expect(result.getByRole('progressbar').getAttribute('aria-valuemin')).toEqual('0');
     expect(result.getByRole('progressbar').getAttribute('aria-valuemax')).toEqual('1');
   });
+  describe('Max', () => {
+    it('sets the proper aria-valuemax when max is negative', () => {
+      const max = -1;
+      const result = render(<ProgressBar value={-2} max={max} />);
+      expect(result.getByRole('progressbar').getAttribute('aria-valuemax')).toEqual('1');
+    });
+    it('sets the proper aria-valuemax when max is zero', () => {
+      const max = 0;
+      const result = render(<ProgressBar value={-5} max={max} />);
+      expect(result.getByRole('progressbar').getAttribute('aria-valuemax')).toEqual('1');
+    });
+    it('sets the proper aria-valuemax when max is valid', () => {
+      const max = 2;
+      const result = render(<ProgressBar value={0} max={max} />);
+      expect(result.getByRole('progressbar').getAttribute('aria-valuemax')).toEqual('2');
+    });
+  });
+  describe('Value', () => {
+    it('sets the proper aria-valuenow when value is greater than max', () => {
+      const value = 23;
+      const max = 10;
+      const result = render(<ProgressBar value={value} max={max} />);
+      expect(result.getByRole('progressbar').getAttribute('aria-valuenow')).toEqual('10');
+    });
+    it('sets the proper aria-valuenow when value is negative', () => {
+      const value = -5;
+      const max = -3;
```

Comment:
Use a valid `max` value since you're not checking for that in this test

```suggestion
      const max = 3;
```